Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.8-slim-buster
```

Comment:
Could you test it works with alpine ?

Reply:
Yup, it built successfully.
The image is half the size with 50 MB vs. 113 MB.